packages/mobile/src/notifications.ts

```diff
@@ -54,51 +43,37 @@ const getPlatformConfiguration = () => {
 class PushNotifications {
   lastId: number
   token: Token | null
-  drawerHelpers: DrawerNavigationHelpers | null
-  bottomTabNavigation: Nullable<BottomTabNavigation>
+  navigation: any | null
```

Comment:
ideally we have a type here, something like `NativeNavigationStackProp<AppTabParamList>` or smthg